packages/client/hooks/useAnimatedSpotlightSource.ts

```diff
@@ -52,6 +54,10 @@ const useAnimatedSpotlightSource = (
     dragIdRef.current = clientTempId()
     // execute mutation after cloning as the mutation will cause reflection height to change
     startDrag(reflectionId, dragIdRef.current)
+    SendClientSegmentEventMutation(atmosphere, 'Opened Spotlight', {
```

Comment:
We could use the `segmentIo` utility function in the `startDraggingReflection` mutation on the server, which is the typical way that we're tracking events in segment. However, as `startDraggingReflection` is firing every second so that the remote animating reflection continues to work even when subscribers refresh the page, I think we should use the client mutation instead as it will fire only once.

Reply:
When a Spotlight event happens, we're sending the following to Segment:
- `viewerId` so we know which users are using the Spotlight
- `reflectionId` so we know if users open the Spotlight with different reflections, if multiple team members open the same reflection, and if the Spotlight is usually opened with the same type of reflection
- `meetingId` so we know in which types of meetings users open the Spotlight. I'd imagine it happens in meetings where there are many reflections, but we'll see

Here, we're not sending `meetingId`. We could pass `meetingId` into the hook or move the mutation to `GroupingKanban` where we close the portal, but as we're tracking the `reflectionId`, we could always track the meeting by looking at the `Closed Spotlight` event, which does send the `meetingId`.

Reply:
why do this on the client vs server?

for example, if we did this on the server (inside the `similarReflections` query) we could capture this info for folks who have adblockers, too.

might also be worthwhile capturing how many reflections there are in the meeting

Reply:
I initially implemented this on the server, but as we're firing `startDraggingReflectionMutation` every second so that the remote Spotlight animation keeps wiggling after a refresh, I figured it would be best to do this once from the client. 

I've added the `reflectionsCount` when closing the Spotlight.